### PR TITLE
Remove texlive and introduce 'git fetch' limit

### PR DESCRIPTION
**Pull request by cajus**
This is an attempt to reduce the required *travis* disk space on the deploy job.

### CODE REVIEW

**Review by johnspackman — DISMISSED** [review]

**Review by cboulanger — DISMISSED** [review]

**Review by level420 — DISMISSED** [review]

**Review by derrell — DISMISSED** [review]

**Review by hkollmann — CHANGES_REQUESTED** [review]
line 77, git clone -q git@github.com:qooxdoo/qooxdoo.github.io.git tmp is a candidate for --depth either....

**Comment by cajus** [comment]
@hkollmann definitely.

**Review by hkollmann — APPROVED** [review]

File: .travis/deploy [hunk 1]
```diff
@@ -74,7 +74,7 @@ git commit -m "Refresh site at ${rev}"
 git push -q upstream HEAD:master
 
 # Do a regular checkout and make a dummy commit
-git clone -q git@github.com:qooxdoo/qooxdoo.github.io.git tmp
+git clone --depth 10 -q git@github.com:qooxdoo/qooxdoo.github.io.git tmp
```
Comment (derrell) [inline .travis/deploy]: I know what it means, but in this particular case, what are these `--depth` options accomplishing? What's so far down the tree that we don't need it, and if we don't need it, why is it there?

**Review by derrell — APPROVED** [review]
See my comment on line 77 (also applies to line 35). If it's just cruft, we should create a separate PR to remove it.

**Review by cboulanger — APPROVED** [review]

**Review by oetiker — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Comment by hkollmann** [comment]
3 days, 5 approvals. Thanks @cajus

**Review by zaucker — COMMENTED** [review]
Sounds ok to me.

**Pull request merged**